field_audit/models.py

```diff
@@ -732,6 +735,11 @@ def delete(self, *, audit_action=AuditAction.RAISE):
         if audit_action is AuditAction.IGNORE:
             return super().delete()
         assert audit_action is AuditAction.AUDIT, audit_action
+
+        from .field_audit import is_audit_enabled
+        if not is_audit_enabled():
+            return super().delete()
```

Comment:
1. Why not do this check above?
```suggestion
        from .field_audit import is_audit_enabled
        if audit_action is AuditAction.IGNORE or not is_audit_enabled():
            return super().delete()
        assert audit_action is AuditAction.AUDIT, audit_action
```

2. Can the import be moved up to the top of the module? I see there are other local imports in this file, but wondering if this particular one is necessary?

These suggestions can be applied in all other changes in this file as well.

Reply:
So that the assert for the AuditAction is still applied even if auditing is disabled. 

My logic was that if you disabled auditing in tests you'd still want to know if this validation failed.

But for other use cases I guess it might not make sense.

Reply:
Looking again, the keyword validation is done in the decorator so it would be fine to combine these two.